components/Footer.js

```diff
@@ -1,9 +1,7 @@
-import styles from '../styles/Home.module.css'
-
 const Footer = () => {
   return (
     <div>
-      <footer className={styles.footer}></footer>
+      <footer /*className={styles.footer}*/></footer>
```

Comment:
Why is this commented out? Should it be here at all?

Reply:
I commented this because I was having issues with the import on the tests, but since we will remove this style and use tailwind (I think there is already a ticket for this) I'll put it back

Reply:
Turns out, we don't have that css in this branch anymore so I removed it!